### PR TITLE
Chore: Rename `crate.io` to `cratedb.com`

### PR DESCRIPTION
**Pull request by amotl**
What the title says.

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -96,9 +96,11 @@ Looking for more help? Check out our `support channels`_.
 
 .. _command-line interface: https://en.wikipedia.org/wiki/Command-line_interface
 .. _contribution docs: CONTRIBUTING.rst
-.. _Crate.io: https://crate.io/
+.. _Crate.io: https://cratedb.com/
```
Comment (msbt) [inline README.rst:99]: @amotl this one stays?

Comment (surister) [reply README.rst:99]: **io** for the company and **db** for the product right?

Comment (amotl) [reply README.rst:99]: Yeah, and the statement "cratedb.com is the website of Crate.io" is also true, and correct, no?

Comment (msbt) [reply README.rst:99]: ah yes, good point, imprint confirms this 😄

**Review by surister — APPROVED** [review]

**Pull request merged**